client/src/containers/CampaignPage.js

```diff
@@ -18,6 +22,15 @@ class CampaignPage extends Component {
   componentDidMount() {
     this.props.fetchCampaignById(this.props.params.id);
     this.props.fetchSignatures(this.props.params.id);
+    this.props.fetchApartmentsRequest();
+  }
+
+  componentWillUpdate(nextProps, nextState) {
+    if (this.props.params.id !== nextProps.params.id) {
+      nextProps.fetchCampaignById(nextProps.params.id);
+      nextProps.fetchSignatures(nextProps.params.id);
+      nextProps.fetchApartmentsRequest();
```

Comment:
fetchApartmentsRequests() gets all the campaigns with with no passed in parameters.  its data does not need effect the map (other then potentially populating newly created campaigns while a user is on the page).  

Removing this would save a nearly-useless call to the api